src/main.jl

```diff
@@ -10,9 +10,16 @@ function sim!(P, C, dt)
 end
 
 function sim!(P, C; dt = 0.1ms, duration = 10ms)
-    for t = 0ms:dt:(duration - dt)
-        sim!(P, C, dt)
-    end
+	sized = duration/dt
+	for p in P
+		if hasproperty(p, :spike_raster)
+			p.spike_raster::Vector{Int32} = zeros(trunc(Int, sized))
+		end
+		for t = 0ms:dt:duration#(duration - dt)
```

Comment:
Dead code should just be removed, not commented out.

Reply:
fixed I think